common/djangoapps/custom_settings/tests/test_views.py

```diff
@@ -0,0 +1,110 @@
+from datetime import datetime
```

Comment:
Shouldn't there be a test case that also tests for the 'PUT' request as this view explicitly allows PUT method

`@require_http_methods(("GET", "POST", "PUT"))`

Reply:
As per code no separate test required for PUT. Both PUT and POST have the same code/logic. (which itself is a wrong implementation of PUT)